README.markdown

```diff
@@ -125,6 +125,10 @@ Make sure you have the file `~/.m2/settings.xml`:
           <username></username>
           <password></password>
         </server>
+        <server>
+          <id>github-project-site</id>
```

Comment:
do you actually use the values "github-project-site" and "git" or are those placeholders for values that you fill in?